CRM/Core/Payment.php

```diff
@@ -599,7 +599,7 @@ public function getText($context, $params) {
           return ts('To complete this transaction, click the <strong>Continue</strong> button below.');
         }
         if ($this->_paymentProcessor['billing_mode'] == 4) {
-          return ts('Click the <strong>Continue</strong> button to go to %1, where you will select your payment method and complete the contribution.', [$this->_paymentProcessor['payment_processor_type']]);
+          return ts('Click the <strong>Continue</strong> button to complete the contribution.');
```

Comment:
OK with changing text to: "button to proceed with the contribution" - minor text change